pkg/algorithm/cstorpoolselect/v1alpha1/select.go

```diff
@@ -46,6 +50,13 @@ type policy interface {
 // policy implementation
 type antiAffinityLabel struct {
 	labelSelector string
+	cvrList       cvrListFn
```

Comment:
Please provide comments about this property esp. why we have introduced this? Refer to code at other places.

Reply:
done